src/dstore/dstore_base.c

```diff
@@ -799,3 +757,125 @@ int dstore_pread(struct dstore_obj *obj, off_t offset, size_t count,
 
 	return rc;
 }
+
+int dstore_dealloc_op(struct dstore_obj *obj, struct dstore_io_vec *vec,
+		      struct dstore_io_op **out)
+{
+	int rc;
+
+	rc = dstore_io_op_init_and_submit(obj, vec, out,
+					  DSTORE_IO_OP_FREE);
+
+	log_debug("dstore_dealloc_op (" OBJ_ID_F " <=> %p, "
```

Comment:
No need to write function name here, as per your recent change in other MR I understood line number and function name will be take care by `log_debug`. I know that we have done the same at lot's of place and we have to remove that but let's not add such new entries.

Please remove it whichever API you have modified.

Reply:
Done.